README.md

```diff
@@ -40,10 +41,10 @@ Lowercase letters are typically used to begin variable names.
 
 - testing is the topic our group selected to do in the assignment that we have to write for it.
 
-- How to do unit testing by Sergey. K [https://www.toptal.com/qa/how-to-write-testable-code-and-why-it-matters#:~:text=A%20typical%20unit%20test%20contains,it%20observes%20the%20resulting%20behavior].
+- How to do unit testing by Sergey. K [Unit Testing and Coding: Why Testable Code Matters](https://www.toptal.com/qa/how-to-write-testable-code-and-why-it-matters#:~:text=A%20typical%20unit%20test%20contains).
 - In sergey explination, he thought about what it is and how we are using it and what are the benifits that we can achive by this unit testing.
```

Comment:
This is the correct way to use a markdown hyperlink: The text of the hyperlink (usually the title of the resource you're linking to) goes first inside square brackets, and the link itself goes next inside rounded parentheses. I've fixed a couple of these for you, to demonstrate how it should be done, but you should fix the rest.